magefiles/helm.go

```diff
@@ -0,0 +1,110 @@
+//go:build mage_helm
+
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"strconv"
+	"strings"
+
+	"github.com/magefile/mage/sh"
+	"golang.org/x/xerrors"
+	"gopkg.in/yaml.v3"
+)
+
+const chartFile = "./helm/trivy/Chart.yaml"
+
+func main() {
+	trivyVersion, err := version()
+	if err != nil {
+		log.Fatalf("could not determine Trivy version: %v", err)
+	}
+	input, err := os.ReadFile(chartFile)
+	if err != nil {
+		log.Fatalf("could not find helm chart %s: %v", chartFile, err)
+	}
+	jsonData := map[string]interface{}{}
```

Comment:
nit:
```suggestion
	yamlData := map[string]interface{}{}
```

Reply:
`version` and `appVersion` are default fields - https://helm.sh/docs/topics/charts/#the-chartyaml-file
let's create struct. something like that:
```
type chart struct {
	version    string
	appVersion string
}
```

Reply:
> nit:

done https://github.com/aquasecurity/trivy/pull/7526/commits/5a43735fabd3d5b22cff4f3b6923ebb72edb9e8e

Reply:
> `version` and `appVersion` are default fields - https://helm.sh/docs/topics/charts/#the-chartyaml-file let's create struct. something like that:
> 
> ```
> type chart struct {
> 	version    string
> 	appVersion string
> }
> ```

added https://github.com/aquasecurity/trivy/pull/7526/commits/8d5a183e0c3e9a7ae247e8b9877f660d7a8b3639